qiskit/assembler/assemble_schedules.py

```diff
@@ -144,8 +144,9 @@ def _assemble_experiments(
 def _assemble_instructions(
         schedule: Schedule,
         instruction_converter: InstructionToQobjConverter,
-        run_config: RunConfig
-) -> Tuple[List[PulseQobjInstruction], Dict[str, Command], int]:
+        run_config: RunConfig,
+        user_pulselib: Dict[str, Command]
+) -> Tuple[List[PulseQobjInstruction], int]:
     """Assembles the instructions in a schedule into a list of PulseQobjInstructions and returns
     related metadata that will be assembled into the Qobj configuration.
```

Comment:
Can you update the documentation to say that the input is modified?

Reply:
Is this what you intended?
https://github.com/Qiskit/qiskit-terra/pull/3900/commits/477f6fb68ed958b66754cd6a761a3646ff912615